src/v0/destinations/facebook_pixel/transform.js

```diff
@@ -200,6 +204,42 @@ const handleProduct = (message, categoryToContent, valueFieldIdentifier) => {
   };
 };
 
```

Comment:
Write unit test for `handleSearch` function under `/test/integrations/destinations/facebook_pixel/transform.js`
cc: @saikumarrs

Reply:
Created this PR - https://github.com/rudderlabs/rudder-transformer/pull/1964/files
Can you add more test cases to it with different scenarios ? You can use the same payload & modify the parameter values before executing a test case.